### ISSUE TITLE
Lock statements should be preceded and followed by a blank line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#509]
Lock statements method should be surrounded by a blank line to ease reading (spotting these statements with ease).

Following should report a violation:
```c#
var x = 42
lock (sync)
{
    ...
}
```

While following should NOT report a violation:
```c#
var x = 42

lock (sync)
{
    ...
}
```
